### PR TITLE
db/datagrid-default-theme

### PR DESCRIPTION
**Pull request by sebastianbochan**

### CODE REVIEW

**Review by bre1470 — APPROVED** [review]

File: ts/DataGrid/DataGridDefaults.ts [hunk 1]
```diff
@@ -32,7 +32,7 @@ import type DataGridOptions from './DataGridOptions';
 /** @internal */
 const DataGridDefaults = {
     /** @internal */
-    cellHeight: 25,
+    cellHeight: 49,
```
Comment (karolkolodziej) [inline ts/DataGrid/DataGridDefaults.ts:35]: As I see in the design only the header row should have a more significant height.
image

Also, could you explain why 49?

**Review by karolkolodziej — CHANGES_REQUESTED** [review]

File: ts/DataGrid/DataGridDefaults.ts (hunk 1 above)
Comment (sebastianbochan) [reply ts/DataGrid/DataGridDefaults.ts:35]: 49 is applied only for header's cells.

That works from the design of data-grid, so I did not refactor the core of it.

File: css/datagrid.css [hunk 2]
```diff
@@ -129,9 +129,9 @@
 }
 
 .hc-dg-cell-input {
-    width: calc(100% + 16px);
-    margin: 0 -8px;
-    padding: 0 6px;
+    width: calc(100% + 24px);
```
Comment (pawellysy) [inline css/datagrid.css:132]: Could we replace this `width: calc(...)` with `flex: 1`?

**Review by pawellysy — CHANGES_REQUESTED** [review]
Minor request

File: css/datagrid.css (hunk 2 above)
Comment (sebastianbochan) [reply css/datagrid.css:132]: Not really, it causes problems with width and highlighting input.

**Review by pawellysy — APPROVED** [review]

**Pull request merged**